content/docs/concepts/clusters.mdx

```diff
@@ -0,0 +1,205 @@
+---
+id: clusters
+title: Clusters
+description: This document describes Clusters in Pomerium Zero.
+keywords: [clusters, pomerium zero]
+sidebar_label: Clusters
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Clusters in Pomerium Zero
+
+This document describes Clusters in Pomerium Zero.
+
+## Overview
+
+A **Cluster** represents a single Pomerium deployment managed by Pomerium Zero, our hosted control plane. You can think of a Pomerium cluster as a completely distinct deployment of Pomerium that allows you to effectively shard your deployments for performance, governance, security, or management purposes.
+
+When you deploy a cluster, it gets its own:
+
+- **Cluster identity**
+- **Cluster domain**
+- **Cluster settings**
+- **Routes and Policies**
+- **Wildcard certificates**
+
+### Cluster use cases: Why clusters?
+
+#### All-in-one mode deployment model
+
+Pomerium consists of 4 services:
+
+- Proxy service
+- Authenticate service
+- Authorize service
+- Databroker service
+
+A cluster deploys these services in all-in-one mode, meaning the configuration for each service is managed from a single cluster instance. All inter-component communications are internally connected, with the exception of the Databroker service, which is an external dependency.
+
+#### Multiple cluster instances
+
+You can deploy multiple clusters from a single cluster identity (see [**Cluster identity**](#cluster-identity) below for more information). The ability to deploy multiple cluster instances means you can manage and run completely separate configurations for workloads that may otherwise be incompatible in a single cluster instance.
```

Comment:
This is incorrect: you cannot deploy multiple clusters from a single cluster identity. 
you may, however, deploy multiple cluster _replicas_ from a single cluster identity. They, however, would share all the cluster configuration. 

> you can manage and run completely separate configurations for workloads that may otherwise be incompatible in a single cluster instance.

what is described is multiple clusters, not multiple cluster instances.